themes/SuiteP/css/suitep-base/editview.scss

```diff
@@ -2037,6 +2038,7 @@ ul.clickMenu.subpanel.records li ul.subnav li a:hover {
 
 ul.clickMenu ul.subnav-sub li a:hover {
   background-color: $list-view-action-menu-link-bg;
+  color: $navbar-link-color;
```

Comment:
Hi @cameronblaikie, is there a reason for using `$navbar-`?

This is file is common to all themes : Dawn, Noon, Day, Night, etc. And the navbar-colors will change in all of those. Which means that it may not match what we expect to get here.

Maybe best to have a specific variable for this?